bigtable/bigtable.go

```diff
@@ -132,6 +133,11 @@ func NewClientWithConfig(ctx context.Context, project, instance string, config C
 		option.WithGRPCDialOption(grpc.WithDefaultCallOptions(grpc.MaxCallSendMsgSize(1<<28), grpc.MaxCallRecvMsgSize(1<<28))),
 	)
 
+	if useDirectAccess, _ := strconv.ParseBool(os.Getenv("CBT_ENABLE_DIRECTPATH")); useDirectAccess {
+		enableDirectAccess = true
+		o = append(o, internaloption.EnableDirectPath(true), internaloption.EnableDirectPathXds())
+	}
```

Comment:
```suggestion

	enableDirectAccess, _ := strconv.ParseBool(os.Getenv("BIGTABLE_ENABLE_DIRECTPATH"))
	if enableDirectAccess {
		o = append(o, internaloption.EnableDirectPath(true), internaloption.EnableDirectPathXds())
	}
```

and then use enableDirectAccess after line 185
i.e.

return &Client{
 ....
enableDirectAccess: enableDirectAccess
}, nil